lib/sles4sap_publiccloud.pm

```diff
@@ -667,4 +673,96 @@ sub create_hana_vars_section {
     return (\%hana_vars);
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online otherwise 0 offline.
+
+=back
+=cut
+
+sub get_hana_database_info {
+    my ($self, $password_db, $instance_id) = @_;
+    my $hana_database_online = '"HDB","HDB-$instance_id","YES","ACTIVE","","","DEFAULT","?"';
```

Comment:
Seems `$hana_database_online` variable is not used at all. Possibly delete it.

Reply:
thx